app/views/purchases/index.html.erb

```diff
@@ -20,19 +20,22 @@
           <th>Payee</th>
           <th>Category</th>
           <th>Purchaser</th>
+          <th>Purchasee</th>
         </tr>
       </thead>
       <tbody>
 
-        <% @purchases.each do |purchase| %>
+        <% @purchases.order(:date).reverse_order.each do |purchase| %>
```

Comment:
I think this slipped by in your merge conflict resolution - you changed the ordering in the controller now :)

Reply:
Nice catch!